src/@apollo/client/link/core/ApolloClient__Link_Core_Types.re

```diff
@@ -38,12 +39,24 @@ module Operation = {
       variables: Js.Json.t,
       operationName: string,
       extensions: Js.Json.t,
-      setContext: Js.Json.t => Js.Json.t,
-      getContext: unit => Js.Json.t,
+      setContext: useMethodFunctionInThisModuleInstead,
+      getContext: useMethodFunctionInThisModuleInstead,
```

Comment:
I rather like this pattern of stubbing out the methods with an abstract type. It at least allows you see on hover what is available.